src/models/mcawsModels.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Sequelize = require('sequelize');
```

Comment:
before we decide on Sequelize, have we done evaluation on Sequelize? Is it documented?

Reply:
no any document but  Sequelize is more easy to use .  and based on internet people are more emphasize to use Sequelize than waterLine. 
and also its independent of sails api code so i thought we can use this orm

Reply:
This means we end up creating two different versions of models, one in waterline and other in sequelize. Why do we require sails independent code is what my question is?

Reply:
we wanted get away from Sails framework and we have never used Sails framework for micro services development. Microservices are written in plain nodejs(no framework).

Reply:
but we need an ORM, we cannot make direct SQL calls.